microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -354,7 +354,7 @@ See also:
 <!-- ------------------------------ -->
 #### Xbox, HoloLens, and XAML limitations
 
```

Comment:
@nishitha-burman -- removed "variant" here and next file.  ok?

Reply:
restored in 2 files

Reply:
Let's reword this section to:
Title: XAML Limitation
XAML Island support requires additional work and may be considered for future releases.